spec/mongoid/delorean/trackable_spec.rb

```diff
@@ -1,24 +1,22 @@
 require 'spec_helper'
```

Comment:
Does this file have any non-stylistic changes?

Reply:
no functional changes. only rubocop fixes. most of the specs are carried forward